client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -58,13 +71,9 @@ export default protectForm( React.createClass( {
 		return settings;
 	},
 
-	isCodeValid( code ) {
-		return ! code || code.match( /^UA-\d+-\d+$/i );
-	},
-
 	handleCodeChange( event ) {
 		const code = event.target.value;
-		const isCodeValid = this.isCodeValid( code );
+		const isCodeValid = googleAnalyticsCodeValidator( code );
```

Comment:
I like the format of the new function but I think the name actually made better sense before. What about naming it `googleAnalyticsCodeIsValid`?